src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final double DEFAULT_RADIUS = 10;
+    private final FigureType[] figureType = FigureType.values();

```

Comment:
make this variable local

Reply:
I do not understand the difference and the reason for making a local or global variable

Reply:
you should make variables global, so you won't need to create every time new instances of these classes and save memory

Reply:
with such questions to jv_chat, it will be faster